pkg/cmd/describe/origin/origin.go

```diff
@@ -1,51 +1,78 @@
-package describe
+package origins
 
 import (
 	"context"
 	"encoding/json"
 	"fmt"
 	"path/filepath"
+	"strconv"
 
 	"github.com/fatih/color"
+	"go.uber.org/zap"
 
 	"github.com/MakeNowJust/heredoc"
 	"github.com/MaxwelMazur/tablecli"
-	msg "github.com/aziontech/azion-cli/messages/origins"
+	msg "github.com/aziontech/azion-cli/messages/describe/origin"
 
 	api "github.com/aziontech/azion-cli/pkg/api/origin"
 	"github.com/aziontech/azion-cli/pkg/cmdutil"
 	"github.com/aziontech/azion-cli/pkg/contracts"
+	"github.com/aziontech/azion-cli/pkg/logger"
 	"github.com/aziontech/azion-cli/utils"
 	sdk "github.com/aziontech/azionapi-go-sdk/edgeapplications"
 	"github.com/spf13/cobra"
 )
 
+var exemplo string = heredoc.Doc(`
+	$ azion origins describe --application-id 1673635839 --origin-key 0000000-00000000-00a0a00s0as0-000000
+	$ azion origins describe --application-id 1673635839 --origin-key 0000000-00000000-00a0a00s0as0-000000 --format json
+	$ azion origins describe --application-id 1673635839 --origin-key 0000000-00000000-00a0a00s0as0-000000 --out "./tmp/test.json" --format json
+	`)
+
 var (
 	applicationID int64
-	originID      int64
+	originKey     string
 )
 
 func NewCmd(f *cmdutil.Factory) *cobra.Command {
 	opts := &contracts.DescribeOptions{}
 	cmd := &cobra.Command{
-		Use:           msg.OriginsDescribeUsage,
-		Short:         msg.OriginsDescribeShortDescription,
-		Long:          msg.OriginsDescribeLongDescription,
+		Use:           msg.Usage,
+		Short:         msg.ShortDescription,
+		Long:          msg.LongDescription,
 		SilenceUsage:  true,
 		SilenceErrors: true,
-		Example: heredoc.Doc(`
-      $ azion origins describe --application-id 1673635839 --origin-id 31223
-      $ azion origins describe --application-id 1673635839 --origin-id 31223--format json
-      $ azion origins describe --application-id 1673635839 --origin-id 31223--out "./tmp/test.json" --format json
-    `),
+		Example:       exemplo,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if !cmd.Flags().Changed("application-id") || !cmd.Flags().Changed("origin-id") {
-				return msg.ErrorMissingArguments
+			if !cmd.Flags().Changed("application-id") {
+				answers, err := utils.AskInput("What is the ID of the Edge Application?")
```

Comment:
Put this input message inside a variable in the messages file
Also:
"What is the ID of the Edge Application this origin is linked to?"